dockers/docker-base-stretch/Dockerfile.j2

```diff
@@ -36,6 +36,10 @@ COPY ["sources.list", "/etc/apt/sources.list"]
 COPY ["no_install_recommend_suggest", "/etc/apt/apt.conf.d"]
 COPY ["no-check-valid-until", "/etc/apt/apt.conf.d"]
 
+{% if CONFIGURED_ARCH == "armhf" or CONFIGURED_ARCH == "arm64" %}
+RUN apt-get clean
+RUN echo "deb http://ftp.debian.org/debian stretch-backports main" | tee /etc/apt/sources.list.d/backports.list
```

Comment:
can u check backports is really needed or not

Reply:
this is needed. otherwise getting this error
Version '5:5.0.3-3~bpo9+2' for 'redis-tools' was not found

Reply:
now redis-tools is part of sonic-storage, so can  we revert it?

Reply:
changed redis version to redis-tools_6.0.6-1~bpo10 from sonic-storage